docs/user/install.rst

```diff
@@ -65,26 +65,33 @@ these issues by setting additional Clang C compiler flags as follows:
 
     $ export CFLAGS="-Qunused-arguments -Wno-unused-function"
 
+Dependencies
+------------
+
+Falcon depends on `six` and `python-mimeparse`. `python-mimeparse` is a
+better-maintained fork of the similarly named `mimeparse` project.
+Normally the correct package will be selected by Falcon's ``setup.py``.
+However, if you are using an alternate strategy to manage dependencies,
+please take care to install the correct package in order to avoid
+errors.
 
 WSGI Server
 -----------
 
-Falcon speaks WSGI. If you want to actually serve a Falcon app, you will
-want a good WSGI server. Gunicorn and uWSGI are some of the more popular
-ones out there, but anything that can load a WSGI app will do. Gevent is
-an async library that works well with both Gunicorn and uWSGI.
+Falcon speaks WSGI, and so in order to serve a Falcon app, you will
+need a WSGI server. Gunicorn and uWSGI are some of the more popular
+ones out there, but anything that can load a WSGI app will do.
 
 .. code:: bash
 
-    $ pip install gevent [gunicorn|uwsgi]
-
+    $ pip install [gunicorn|uwsgi]
 
 Source Code
 -----------
 
 Falcon `lives on GitHub <https://github.com/racker/falcon>`_, making the
```

Comment:
I know this isn't under changes in review, but we should probably change this one as well.